pkg/controllers/addons_controller.go

```diff
@@ -270,30 +163,30 @@ func (r *AddonsLayerReconciler) update(ctx context.Context, log logr.Logger,
 }
 
 /*
-func (r *AddonsLayerReconciler) gitRepositorySource(o handler.MapObject) []ctrl.Request {
+func (r *AddonsLayerReconciler) sourceController(o handler.MapObject) []ctrl.Request {
 	//ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
 	//defer cancel()
 
 	return []ctrl.Request{
 		{
 			NamespacedName: types.NamespacedName{
-				//Name:      sourcev1.GitRepository.Name,
-				//Namespace: sourcev1.GitRepository.Namespace,
+				Name:      sourcev1.GitRepository.Name,
+				Namespace: sourcev1.GitRepository.Namespace,
 			},
 		},
 	}
 }
 */
-
 // SetupWithManager is used to setup the controller
 func (r *AddonsLayerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	addonsLayer := &kraanv1alpha1.AddonsLayer{}
 	_, err := ctrl.NewControllerManagedBy(mgr).
 		For(addonsLayer).
-		/*		Watches(
```

Comment:
@paulcarlton-ww We should add Owns(&hr.HelmRelease) right? This would trigger reconcile of the addon layer whenever the helm releases it owns changes.